protocols/identify/src/behaviour.rs

```diff
@@ -156,12 +160,29 @@ pub struct Config {
 
 impl Config {
     /// Creates a new configuration for the identify [`Behaviour`] that
-    /// advertises the given protocol version and public key.
-    pub fn new(protocol_version: String, local_public_key: PublicKey) -> Self {
+    /// advertises the given protocol version and public key.  
+    /// Use [`new_with_keypair`](Config::new_with_keypair) for `signedPeerRecord` support.  
+    pub fn new(protocol_version: String, public_key: PublicKey) -> Self {
         Self {
             protocol_version,
             agent_version: format!("rust-libp2p/{}", env!("CARGO_PKG_VERSION")),
-            local_public_key,
+            local_key: public_key.into(),
```

Comment:
```suggestion
    pub fn new(protocol_version: String, local_public_key: PublicKey) -> Self {
        Self {
            protocol_version,
            agent_version: format!("rust-libp2p/{}", env!("CARGO_PKG_VERSION")),
            local_key: local_public_key.into(),
```
Stick with `local_public_key` so it's consistent with `local_keypair` below?

Reply:
I would like to keep existing API unchanged, so I'm changing `local_keypair` to `keypair` below.

Reply:
But isn't it called `local_public_key` in the existing API?